macro-fullcalendar-ui/src/main/resources/Calendar/Macro.xml

```diff
@@ -288,6 +287,24 @@ or
 /************************************
  * Configure RequireJS to use jQuery-UI
  ************************************/
+// The configuration of this require block needs to be present here because of the Chrome problem: when reloading the
+// Chrome tab the resources will not be loaded.
```

Comment:
@ignatescustefan this comment is not very precise. The require block below has some dependencies ('jquery', 'jqueryui', 'fullcalendar', 'gcal') and the path to load these dependencies must be set somewhere **before** the require block is executed, otherwise it fails. Before your change the require configuration was done in ``Calendar.FullCalendar`` which was loaded using ``$xwiki.jsx.use()``. The problem was that there was no **deterministic** order between loading ``Calendar.FullCalendar`` and executing this require block. ``Calendar.FullCalendar`` could very easily be loaded **after** the require block that uses the configured modules. In order to fix this you had two options:
* move the require configuration before the require block, which you did. I hope this doesn't break anything
* wrap the existing require block in another require block that loads the require configuration:
```
require(new XWiki.Document('FullCalendar', 'Calendar').getURL('jsx'), function() {
  require(['jquery', 'jqueryui', 'fullcalendar', 'gcal'], function ($) {
    ...
  });
});
```